projects/1.18/assets/tiny-pipes/tinypipes/lang/zh_cn.json

```
@@ -0,0 +1,24 @@
+{
+  "itemGroup.tinypipes": "迷你管道",
+  "item.tinypipes.item_pipe": "迷你物品管道",
+  "message.item.tinypipes.item_pipe": "可以在容器之间传输物品。\n手持红石扳手右击改变输入输出面。\n空手右击以打开GUI。",
+  "item.tinypipes.item_filter_pipe": "迷你物品过滤管道",
+  "message.item.tinypipes.item_filter_pipe": "可以在容器之间传输指定的物品。\n空手右击以打开过滤器。\n手持红石扳手右击改变输入输出面。",
+  "item.tinypipes.fluid_pipe": "迷你流体管道",
+  "message.item.tinypipes.fluid_pipe": "可以在流体储罐之间传输流体。\n手持红石扳手右击改变输入输出面。\n空手右击以打开GUI。",
+  "item.tinypipes.fluid_filter_pipe": "迷你流体过滤管道",
+  "message.item.tinypipes.fluid_filter_pipe": "可以在流体储罐之间传输指定的流体。\n空手右击可使用桶装液体设置过滤器。\n手持红石扳手右击改变输入输出面。",
+  "item.tinypipes.energy_pipe": "迷你能量管道",
+  "message.item.tinypipes.energy_pipe": "可以把FE从输入端传输到接收端。\n手持红石扳手右击改变输入输出面。\n空手右击可使用流体桶设置过滤器。",
+  "item.tinypipes.redstone_pipe": "迷你红石能量管道",

```

```suggestion
  "item.tinypipes.redstone_pipe": "迷你红石管道",
```
既然有`能量`管道，似乎这里就是单纯传输红石信号的管道。